### PR TITLE
fixed the ui for the tax rates and tax category

### PR DESCRIPTION
**Pull request by deepaksinghgusain227**
#5602 
Fixed ui issue in tax rates in mobile view

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 1]
```diff
@@ -912,7 +912,7 @@
             'create'          => 'Create',
             'add-title'       => 'Create Tax Category',
             'edit-title'      => 'Edit Tax Category',
-            'save-btn-title'  => 'Save Tax Category',
+            'save-btn-title'  => 'Save',
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:915]: If the translation is being changed, then make sure all files present in the lang folder should be present here.

Comment (deepaksinghgusain227) [reply packages/Webkul/Admin/src/Resources/lang/en/app.php:915]: means we have to override all the lang files?

**Review by ghermans — CHANGES_REQUESTED** [review]
As we discussed yesterday translations should not be changed because the word size is to big and overlap in mobile view, this should be handled from scss @devansh-webkul can you follow up this one together with @deepaksinghgusain227 ?

**Comment by Sanjaybhattwebkul** [comment]
@deepaksinghgusain227 
**Not working** properly. Still getting **the same issue** .
Please find the screenshot [Here](https://prnt.sc/26dgyw1)

**Comment by deepaksinghgusain227** [comment]
creating new pr for that

**Pull request closed without merging**